typescript/sdk/src/ism/read.ts

```diff
@@ -69,8 +70,16 @@ export class EvmIsmReader implements IsmReader {
     this.provider = this.multiProvider.getProvider(chain);
   }
 
-  public static stringifyConfig(config: IsmConfig, space?: number): string {
-    return JSON.stringify(config, ethersBigNumberSerializer, space);
+  public static stringifyConfig(
```

Comment:
You can DRY this up with the other stringify method by using a generic for the config type. E.g.

`stringifyConfig<C>( config:C...)`

Reply:
looks like the type isn't used at all so could also make the config type `object`

Reply:
ditto on deduplicating

Reply:
I think I'll probably do that by moving this to utils - would be useful for core/warp modules later too